cmd/mistryd/job.go

```diff
@@ -32,9 +32,7 @@ type Job struct {
 	ID string
 
 	// user-provided
-	Project string
-	Params  types.Params
-	Group   string
```

Comment:
`Job` is a core struct and should not get everything that a `JobRequest` has. In the future we might add more fields and methods to `JobRequest` but we don't want to add them here.

On the other hand, I see why you chose to do so and perhaps we should extract those common fields into yet another type called `JobInfo` maybe or something like that.

Anyway, I think we should leave this for another patch.